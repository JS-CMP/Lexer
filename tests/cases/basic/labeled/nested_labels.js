outer: for (var i = 0; i < 3; i++) {
    inner: for (var j = 0; j < 3; j++) {
        if (i === 1 && j === 1) {
            break outer;
        }
        console.log(i, j);
    }
}

