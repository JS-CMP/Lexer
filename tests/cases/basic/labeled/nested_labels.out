#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {
    JS::Any i, j;
outer:
        for (i = JS::Any(0); i < JS::Any(3); i++) 
    {
inner:
                for (j = JS::Any(0); j < JS::Any(3); j++) 
        {
            if (i strictEq JS::Any(1) && j strictEq JS::Any(1)) 
            {
                break /* label: ; JS_CMP does not support labeled break for now */;
            }
            console[u"log"](i, j);
        }
    }
    return 0;
}
